pkg/process/util/netns.go

```diff
@@ -59,7 +62,9 @@ func GetNetNamespaces(procRoot string) ([]netns.NsHandle, error) {
 	err := WithAllProcs(procRoot, func(pid int) error {
 		ns, err := netns.GetFromPath(path.Join(procRoot, fmt.Sprintf("%d/ns/net", pid)))
 		if err != nil {
-			log.Errorf("error while reading %s: %s", path.Join(procRoot, fmt.Sprintf("%d/ns/net", pid)), err)
+			if !errors.Is(err, os.ErrNotExist) && !errors.Is(err, unix.ENOENT) {
+				log.Errorf("error while reading %s: %s", path.Join(procRoot, fmt.Sprintf("%d/ns/net", pid)), err)
```

Comment:
nit: Is this function gonna run a lot? If so `+` might be better than `Sprintf` but if not please ignore me.

Reply:
This log should not be hit often.